.github/workflows/continuous-integration.yml

```diff
@@ -102,19 +66,26 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+        continue-on-error: ${{ matrix.experimental }}
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -r requirements.txt
+          # All dependencies but markdown-it-py
+          python -m pip install nbformat pyyaml toml
           python -m pip install -r requirements-dev.txt
-          # install sphinx_gallery and matplotlib if available (may not work on pypy)
+          # install sphinx_gallery and matplotlib if available
           python -m pip install sphinx_gallery~=0.7.0 || true
           python -m pip install jupyter-fs || true
+      - name: Install markdown-it-py
+        if: ${{ matrix.markdown-it-py-version }}
+        run: python -m pip install markdown-it-py==${{ matrix.markdown-it-py-version }}
```

Comment:
I think this one is better as `~=`. There can be bugs in the `3.0.0` version and if `~=` is used, it will pick up the latest version

Reply:
Agreed (I am afraid I just understood how `~=` works). That's part of the next PR. Thank you for your patience!